flow/aria.js

```diff
@@ -344,7 +344,7 @@ type ARIARoleRelationConcept = {
 
 type ARIARoleRelationConceptAttribute = {
   name: string,
-  value?: string | number,
+  values?: string | number,
```

Comment:
It is named `value` at the moment as far as I can tell: https://github.com/A11yance/aria-query/blob/e803d941200ddec6528195abe9f4cbce116462dd/src/etc/roles/literal/checkboxRole.js#L28

Though it's weird that type-checking is not catching this. Probably because we don't use exact types at this point. 

Does type-checking still pass if you convert this to an exact type: `{} -> {| values?: string | number |}`.

Reply:
Omg, I mistook the names, haha. I thought I was changing the `ARIAPropertyDefinition` (I did it through the Github CLI and didn't realize it). Now I checked that `ARIAPropertyDefinition` was already fixed by # 74. Sorry for the confusion 😆